src/README.html

```diff
@@ -925,13 +926,159 @@ <h4 id="inline">
   to make finer distinctions between, e.g. different types of code snippets.
   A future revision of the site-wide styles may incorporate some of the more
   common distinctions.
+  
+  <section>
+  <h4 id="char_ref_template"><span class="secno">2.3.7</span> Character naming template</h4>
+
+
+
+	<div class="xref" style="font-size: 80%; padding: 1em; background-color: #fbf4e0;border-radius: 5px;padding: .25em 1em;"><span class="seealso">See also</span>
+	<ul>
+	   <li>[<a href="https://www.w3.org/International/i18n-activity/guidelines/editing#codepoints">I18N Editing Guidelines</a>].</li>
+	   <li>[<a href="https://www.w3.org/TR/international-specs#char_ref_template"><cite>Best Practices for International Specifications</cite></a>].</li>
+	</ul>
+	</div>
+	
+<p>Use of the character template used in Internationalization specifications is recommended. For most characters, the template looks like this:</p>
+
+<pre>
+&lt;span class="codepoint" translate="no">&lt;bdi lang="??">&#xXXXX;&lt;/bdi> 
+&lt;code class="uname">U+XXXX Unicode_character_name&lt;/span>&lt;/code>
+</pre>
+
+<p>For invisible characters (such as control characters) or for whitespace, the character and its surrounding <code translate="no" class="kw">bdi</code> element should be omitted:</p>
+
+<pre>
+&lt;span class="codepoint" translate="no">
+&lt;code class="uname">U+XXXX Unicode_character_name&lt;/span>&lt;/code>
+</pre>
+
+<aside class="example" title="Example of a character reference">
+<p>Filling in the above template like this:</p>
+
+<pre>
+&lt;span class="codepoint" translate="no">
+   &lt;bdi lang="fr">&#x00E9;&lt;/bdi>&lt;code class="uname">U+00E9 LATIN SMALL LETTER E WITH ACUTE&lt;/code>
+&lt;/span>
+</pre>
+
+<p>Produces output in the page like this: <span class="codepoint" translate="no"><bdi lang="fr">é</bdi><code class="uname">U+00E9 LATIN SMALL LETTER E WITH ACUTE</code></span>.</p>
+</aside>
+
+<p>The <code translate="no" class="kw">bdi</code> element is used to ensure that example characters that are right-to-left do not interfere with the layout of the page. Although the elements are shown on separate lines in this section (for clarity), you do not need to include line breaks or a space between the closing <code translate="no" class="kw">bdi</code> and the following <code translate="no" class="kw">span</code> element: spacing and presentation is controlled by styling.</p>
+
+<p>The <code translate="no" class="kw">lang</code> attribute should be filled in appropriately to get the correct font selection for a given context. Examples in East Asian languages (such as Chinese, Japanese, or Korean) or in the Arabic script can sometimes require greater care in choosing a language tag.</p>
```

Comment:
Not sure what you mean here. Do you want to change this text or is this about something else?

Reply:
I think there needs to be a paragraph to remind people using the markup that they will need to use appropriate styling in their style sheet, where necessary, to correctly render the character in the bdi element in terms of font-family and font-size.

Reply:
Please suggest something?

Reply:
Make that an extension of the above para, rather than a separate para.  Perhaps add as the second sentence:

> Your style sheet may need to indicate the appropriate font-family and font-size for characters in a given language.

Reply:
I adapted that thusly:

> In some cases, it might also be necessary for you to adjust the <kbd translate="no">font-family</kbd> and/or <kbd translate="no">font-size</kbd> in your own style sheet for characters or character sequences in a given language.

Reply:
I think that text sounds good, but i'm not seeing it in the diff.